### ISSUE TITLE
Standardisation of sound sets

### ISSUE THREAD

**Issue by Moburma** [swfans/syndwarsfx-sfx#10]
I assume we want to normalise the sound sets, so they all contain the same sounds at a base level? I'm thinking here specifically of the mech sound, which was clearly meant to be in the game, and seems to have just been missed off the English release (and some of the others). Should that be added so it's in all sets?

**Comment by mefistotelis** [comment]
Yes.

We can also have "flavors" of sound sets.. like "standard" for the normal one, "beta" with additional sound which were removed later (like the cop speeches and blimp wind up/wind down), and "asian" with some weapon sounds replaced.

Though for the flavors, I haven't thought about this before. The file names would have to include a flavor either as a number or 3-char shortcut. And instead of 3, we could provide just 2 sound sets - that would simplify thing while making the difference easier to recognize.

**Comment by Moburma** [comment]
Yeah, it gets a bit tricky. As a baseline there are two clearly identifiable sets of sounds that people will be looking for:

- Version 1 - This is the standard English set. Missing the mech sound, which needs to be fixed
- Version 2 - This is the revised version which came out a few days after the English sounds were finalised. This has some different weapon and other sounds. Known best as the French/German version

There is also an earlier set of sounds wrongly included with the Japanese sound.dat (I need to update this on TCRF as this isn't written up properly there), but no one will be actively looking for these (e.g. @geist22 was looking for Version 2 sounds as above as he grew up with that version) as no public release of the game used them.

The beta stuff is also difficult - I've been playing with the extra police sounds for a while, but I see why they cut them now, it's not just they are not as good performance quality as the ones they kept in (which is true - they are clearly not), but with so many to choose from the game spams LOADS of different lines when lots of cops are on screen and it's _really_ annoying! I assume there is some kind of limit on how many times a sound can repeat, but with so many to choose from it's really grating and artificial sounding.
